src/ISeabotConfig.ts

```diff
@@ -0,0 +1,15 @@
+export default interface ISeabotConfig {
+    userVoiceChannels: {
+        groupId: string;
+        triggerChannelId: string;
+    };
+    autoDeleteMessagesInChannels: Array<string> | undefined;
```

Comment:
I'm noodling this one - technically we only have one channel that uses this, but it may be cool in the future to have other channels (like voice-text type channels) that follow a similar pattern with X messages or Y elapsed time since last message)

Reply:
Doesn't hurt to leave the implementation open, for now. I've added control over deletion frequency to cover other situations. We can add X messages as an option if needed.

Reply:
Added X messages as an option.